examples/skills.py

```diff
@@ -856,7 +834,7 @@ def __init__(self, agent):
         self.terminate = False
 
     def update(self):
-        command = self.rob.getCachedObserve('getChat')
+        command = self.rob.getCachedObserve('getChat')[-1][0]
```

Comment:
The first chat message might be something useless  like "give" or "teleport", i would rather check all the messages here.

Reply:
So your proposition is to loop through all commads and executing first one which looks like command (e.g. contains "get something" or "terminate")?

Reply:
Yes

Reply:
@noskill  Done

Reply:
Yep, but that's not the problem since we are looping through reversed list